pom.xml

```diff
@@ -20,7 +20,7 @@
     <mockito.version>1.9.5</mockito.version>
     <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
     <tinybundles.version>2.0.0</tinybundles.version>
-    <pentaho-osgi-utils.version>${project.version}</pentaho-osgi-utils.version>
+    <pentaho-osgi-utils.version>9.3.0.6-786</pentaho-osgi-utils.version>
```

Comment:
As per [this](https://github.com/pentaho/pentaho-osgi-bundles/pull/486/files#diff-72f4d8e15a3876a220e2eb29a4b4b26f3477e12bf886303a31792ffdba2e1353R14), shouldn't this be `9.3.0.6-C284?